src/crypto.rs

```diff
@@ -231,13 +234,9 @@ impl CryptoManager {
     pub fn calculate_mac(&self, msg: &[u8]) -> Result<[u8; 32]> {
         generichash_quick(msg, Some(&self.mac_key))
     }
-
-    pub fn calculate_hash(&self, msg: &[u8]) -> Result<[u8; 32]> {
-        generichash_quick(msg, None)
-    }
```

Comment:
This and `verify_detached()` are only used in tests and generate unused code warnings, so I decided to remove them entirely. Ok, or should I put an `#[allow]` on them instead?

Reply:
It's fine.